### PR TITLE
Text Sets: Update text set using Open Sans Condensed

### LINKED ISSUE #10828: Text Sets: Update text set using Open Sans Condensed
## Feature Description

This is a follow-up to #10819 and e6dc2e95961688d3935a9849587ae50426b0c0f2

The “Eyeshadow Makeup” text set uses the Open Sans Condensed font which has since been dropped from Google Fonts.

Screenshot 2022-03-07 at 17 52 26

We need to replace it with something different, perhaps just Open Sans, or Roboto Condensed.

Can't use Open Sans with tweaks since we don't support  variable fonts (see #1260)

**Update:** We're replacing the Open Sans Condensed font usage as follows:

**Old:** Open Sans Condensed, Regular, font size 24, letter-spacing 4%

**New:** Roboto Condensed, Light, font size 23, letter-spacing 2%. Plus increasing Y position by 5

<table>
<tr>
	<td> Before
	<td> After
<tr>
	<td>before
	<td>after
</table>

Code that needs to be changed:

https://github.com/GoogleForCreators/web-stories-wp/blob/4cbc5bc0759e01653fed9e5c33fed3d55b59bc62/packages/text-sets/src/raw/step.json#L1876-L1927

## Acceptance Criteria

* [ ] Update text set as per the above instructions
* [ ] Unskip test from #10830

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Replace usage Open Sans Condensed with Roboto Condensed. Tweak styles to make look close as possible. 

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #10828

### CODE REVIEW

File: packages/text-sets/src/raw/step.json [hunk 1]
```diff
@@ -1880,7 +1880,7 @@
           "lockAspectRatio": false,
           "backgroundTextMode": "NONE",
           "font": {
-            "family": "Open Sans Condensed",
+            "family": "Roboto Condensed",
```
Comment (timarney) [inline packages/text-sets/src/raw/step.json:1883]: In the issue I see:

> Roboto Condensed, Light  

If that meant to be  Roboto Condensed Light  

Does the font weight need to be updated to 300 ?

Comment (swissspidy) [reply packages/text-sets/src/raw/step.json:1883]: > Does the font weight need to be updated to 300 ?

Yup 👍

**Comment by swissspidy** [comment]
Could you please fill out the PR description template?

Also noting that `I have added a matching Type: XYZ label to the PR` is checked but there are no labels.

File: packages/text-sets/src/raw/step.json [hunk 2]
```diff
@@ -1908,13 +1908,13 @@
...
-          "fontSize": 24,
+          "fontSize": 23,
           "backgroundColor": { "color": { "r": 196, "g": 196, "b": 196 } },
           "lineHeight": 1.3,
           "textAlign": "center",
           "padding": { "locked": true, "horizontal": 0, "vertical": 0 },
           "type": "text",
-          "content": "<span style=\"color: #444; letter-spacing: 0.04em\">Quick Tutorial for Beginners</span>",
+          "content": "<span style=\"color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
           "fontWeight": 400,
```
Comment (spacedmonkey) [inline packages/text-sets/src/raw/step.json]:
```suggestion
          "fontWeight": 300,
```

Comment (spacedmonkey) [reply packages/text-sets/src/raw/step.json]: Is this what you recommend - https://github.com/GoogleForCreators/web-stories-wp/pull/10886/files#r825824820 ?

Comment (swissspidy) [reply packages/text-sets/src/raw/step.json]: `font-weight` is actually part of `content`, e.g.

```
"content": "<span style=\"font-weight: 300; color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
```

I'm not sure where `fontWeight` is coming from 🤔  We might need to remove that. I'll look into it.

Comment (swissspidy) [reply packages/text-sets/src/raw/step.json]: See #10922 for removing those.

**Review by swissspidy — CHANGES_REQUESTED** [review]
`font-weight: 300` is missing

File: packages/text-sets/src/raw/step.json [hunk 3]
```diff
@@ -1908,13 +1908,13 @@
...
           },
-          "fontSize": 24,
+          "fontSize": 23,
           "backgroundColor": { "color": { "r": 196, "g": 196, "b": 196 } },
           "lineHeight": 1.3,
           "textAlign": "center",
           "padding": { "locked": true, "horizontal": 0, "vertical": 0 },
           "type": "text",
-          "content": "<span style=\"color: #444; letter-spacing: 0.04em\">Quick Tutorial for Beginners</span>",
+          "content": "<span style=\"color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
```
Comment (spacedmonkey) [inline packages/text-sets/src/raw/step.json]:
```suggestion
          "content": "<span style=\"font-weight: 300; color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
```

**Review by timarney — APPROVED** [review]

**Review by swissspidy — APPROVED** [review]
Works as expected

**Pull request merged**